cg/meta/report/rnafusion.py

```diff
@@ -71,8 +71,8 @@ def get_genome_build(self, analysis_metadata: AnalysisModel) -> str:
     def get_report_accreditation(
```

Comment:
```suggestion
    def is_accredited(
```
The current function name indicates an object is returned, although it is just a bool that is returned.

Reply:
Sounds good!